### DISCUSSION TITLE
Handle proximity contacts for people without access to mobile phones: support tokens

### DISCUSSION THREAD

**Discussion by LeCanardQuoi** [cds-snc/covid-alert-documentation#47] [Ideas] [answered]
# Summary | Résumé

As a user, I wish to track not only my problematic proximity contacts (#1304) with other mobile phone app users (as app adoption rate increases) and with businesses equipped with QR codes (#1328) but also with people not using these particular phones, such as elderly, school children and less fortunate ones. Finally I wish to be notified sooner rather than later via the app rather than the problematic contact tracing, should a propagation be linked to people without access to mobile phones.

# Design detail | Précisions au sujet de la conception

The app would record a token encounter as an event similar to a proximity contact (5 or 15 minutes) and report event in a daily log. The “token contact” event could be temporarily notified by the app to the screen. Should a propagation be linked to that token contact via diagnosed reporting, my app should be similarly notified. The existing Google/Apple ENS platform should provide for such token devices implementation. Existing app should be able to record such a token proximity contact. Specifics of reciprocating token device may however require testing and fine tuning. 

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet

Clearly out of scope of the existing app is the actual “token” implementation (paired with a non-mobile component). It would be preferably sourced “off the shelves” as other countries such as Singapore has embraced this, driving to a 90% app adoption rate as paired with public policies on accessing public places and on making these tokens available for universal access. This enhancement request would be best reviewed as part of a relaunch driving towards higher adoption rates in response to a new virus strain propagating faster, hence requiring a stronger app solution.  And let's remember lessons from #Ebola when reviewing: "Moving quickly is essential" and avoid aiming for perfection before making a move”. A “universal access” solution with limited capabilities could also be pursued with canadian phone companies configuring less expensive phones for such users until tokens become available in "red zone" problem areas.

**Comment by obrien-j** [comment]
In keeping with our newly really release cds-snc/covid-alert-documentation#31 practices, I've moved this up into the covid-alert-documentation repo as our central point for taking input from folks outside the team, and am converting the issue into a Discussion.

**Comment by obrien-j** [comment, marked as answer]
Thanks for the idea @LeCanardQuoi ,

We don't currently have any plans to support 'wearables', but if the technology landscape changes to provide an equivalent solution to the Apple / Google exposure notification framework, it would definitely be worth considering.

**Reply by LeCanardQuoi** [reply]
Mobile phone loaners (on older phones phased out by telcon providers) is probably the easiest way to broaden adoption rates for improved universal access. Battery holding capacity remains however a serious issue on these older phones. QR code scanning too? Perhaps some are more suitable. Mobile phone providers/operators should really know better and should be consulted, if not invited to participate to this discussion repository. Singapore and NZ have deployed token and card solutions. Alberta ABTraceTogether (Singapore cousin) did not attempt trialing tokens, to my knowledge. Apple Watches and fitbit  are also fielding this, out of the GAENS framework; these however would not really help universal access.

**Reply by LeCanardQuoi** [reply]
GAENS BLE cards failing field trial in NZ
"Covid-19: Chris Hipkins confirms CovidCard not practical" https://amp.rnz.co.nz/article/fa96436c-a7b6-4464-859d-472a5aaa9378
Inexpensive mobile phones &loaners likely a better universal access solution in a near future. Singapore token cards remain TBD...
